### PR TITLE
Enable importing single dossiers with votes

### PR DESCRIPTION
**Pull request by njoyard**
This PR adds a new script to import a single dossier with votes (from http://parltrack.euwiki.org/dossier/YYYY/NNNN(XXX)?format=json URLs). The existing import code has been moved to a common lib module in order to enable usage from any script.

The thing works, but uses exactly the same import code as previously existing scripts. I will try to rewrite those to be less DB-intensive soon.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ script:
 - flake8 representatives_votes/ --exclude migrations --ignore E128
 - py.test
 - cat representatives_votes/contrib/parltrack/tests/dossiers_fixture.json | parltrack_import_dossiers
+- cat representatives_votes/contrib/parltrack/tests/single_fixture.json | parltrack_import_single_dossier
```
Comment (jpic) [inline .travis.yml]: Est-ce-qu'on a un use case pour cette commande ?

File: representatives_votes/contrib/parltrack/tests/test_import.py [hunk 2]
```diff
@@ -50,3 +51,14 @@ def test_parltrack_import_votes():
...
     _test_import('votes', import_votes.main)
+
+
+@pytest.mark.django_db
+def test_parltrack_import_single_dossier():
+    for model in (Representative, Dossier, Proposal, Vote):
+        model.objects.all().delete()
+
+    call_command('loaddata', os.path.join(os.path.abspath(
+        parltrack.__path__[0]), 'tests', 'representatives_expected.json'))
```
Comment (jpic) [inline representatives_votes/contrib/parltrack/tests/test_import.py]: Un petit assertNumQueries ? ;)

File: .travis.yml (hunk 1 above)
Comment (njoyard) [inline .travis.yml]: Pas sur, je l'ai mis par homogénéité avec les autres

File: representatives_votes/contrib/parltrack/tests/test_import.py (hunk 2 above)
Comment (njoyard) [inline representatives_votes/contrib/parltrack/tests/test_import.py]: Ca va venir, mais pour l'instant c'est le même code qu'avant qui importe ! Patience :)

File: .travis.yml (hunk 1 above)
Comment (jpic) [inline .travis.yml]: Si on peut s'en passer pour l'instant est-ce-qu'on peut le retirer de la PR ?

Comment (njoyard) [inline .travis.yml]: Oui, je vais le faire

**Comment by jpic** [comment]
I think we could do without the executable command here, but I thought we'd have an entry point to synchronize a Proposal, including download of the JSON data from parltrack ?

**Comment by njoyard** [comment]
I can't find a JSON entry point for proposal downloads on parltrack, is there one ?
Regarding downloads from parltrack, this is done in political_memory for now, that's why there's only the importing part here.

**Comment by jpic** [comment]
Yeah there are urls like /dossier/[reference]?format=json http://parltrack.euwiki.org/dossier/2013/0136%28COD%29?format=json

**Pull request closed without merging**